client/src/App.js

```diff
@@ -4,7 +4,7 @@ import Guideline from './components/layouts/Guideline';
 import Checkout from './components/commonComponents/TabsCheckout';
 import NavBar from './components/commonComponents/NavBar';
 import LandingPage from './components/layouts/LandingPage';
-
+import Error404 from './components/layouts/pageNotFound'
```

Comment:
Should use PageNotFound for the filename here, we usually use this casing to indicate that it's a React component